setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='hm_pyhelper',
-    version='0.13.36',
+    version='0.13.37',
```

Comment:
as none of the my comments will impact the current functionality. Given time critical nature of this change. we can take care of them in a separate ticket.